docs/AboutUs.md

```diff
@@ -9,51 +8,41 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Ho Yin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/lhy-hoyin.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/lhy-hoyin)]
 [[portfolio](team/johndoe.md)]
 
 * Role: Project Advisor
+* Responsibilities: Data
 
-### Jane Doe
+### Jeremy
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jrmckh.png" width="200px">
 
-[[github](http://github.com/johndoe)]
+[[github](http://github.com/JrmCkh)]
 [[portfolio](team/johndoe.md)]
 
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Yu Quan
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/bigheartsmallbrain.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/BigheartSmallbrain)] [[portfolio](team/johndoe.md)]
```

Comment:
Maybe separate it into 2 lines, similar to the rest.
Portfolio need to update in the future. (No actions required)